### ISSUE TITLE
Is husky supposed to be required?

### ISSUE THREAD

**Issue by hildjj** [raineorshine/npm-check-updates#823]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

---------------------------

## Steps to Reproduce

`npm install -g npm-check-updates`

## Current Behavior

```
npm ERR! path /usr/local/lib/node_modules/npm-check-updates
npm ERR! command failed
npm ERR! command sh -c husky install
npm ERR! sh: husky: command not found
```

## Expected Behavior

Clean install.

It looks like husky was added as a devDependency.  I don't *think* you wanted it to be called for end users like myself.

**Comment by raineorshine** [comment]
Thanks! Should be `devDependency` only. Accidentally added it to the `postinstall` script.

**Comment by raineorshine** [comment]
Fixed in `v11.1.3`.
